cumulus/polkadot-parachain/src/command.rs

```diff
@@ -839,6 +871,30 @@ async fn start_node<Network: sc_network::NetworkBackend<Block, Hash>>(
 				.map(|r| r.0),
 		}
 		.map_err(Into::into),
+		Runtime::Omni(consensus) => match consensus {
+			// rococo actually uses aura import and consensus, unlike most system chains that use
+			// relay to aura.
```

Comment:
"relay to aura" is generic that it also works if you just have an aura chain from gensis.

Reply:
yeah, just that it is needlessly checking some `if` statement, so it is a miniscule slower, but it indeed works.

Reply:
Not sure if I understood correctly. Are you suggesting to use `start_generic_aura_lookahead_node` here ?